cmd/skopeo/main_test.go

```diff
@@ -23,7 +24,11 @@ func TestGlobalOptionsNewSystemContext(t *testing.T) {
 	// Default state
 	opts, _ := fakeGlobalOptions(t, []string{})
 	res := opts.newSystemContext()
-	assert.Equal(t, &types.SystemContext{}, res)
+	// User-Agent is set by default.
+	defaultContext := &types.SystemContext{
```

Comment:
(Non-blocking: It would be consistent with the other test to inline this into the `assert.Equal`. This works fine, of course.)

Reply:
Done.